.dotnet/src/Custom/FineTuning/FineTuningJobHyperparameters.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+
+namespace OpenAI.FineTuning;
+
+[CodeGenModel("FineTuningJobHyperparameters")]
+public partial struct FineTuningJobHyperparameters
```

Comment:
Mentioned earlier, but more relevant here anyway: this `FineTuningJobHyperparameters` base is the *output* model on the response's `$.hyperparameters` JSON property; you'll see the generated serialization/deserialization code doesn't know anything about batch_size and learning_rate_multiplier since they aren't part of that response-specific schema type.

We'll want two types, I think:
- CreateFineTuningJobRequestHyperparameters -> FineTuningHyperparameterOptions (request model with all three things)
- FineTuningJobHyperparameters -> FineTuningHyperparameterInfo (response model; the "info" might be contentious and we can discuss naming creativity more broadly)